### PR TITLE
Fix #376 and #377

### LINKED ISSUE #376: django settings ALLOWED_HOSTS = []
/FuzzManager/server/server/settings.py
should have 
ALLOWED_HOSTS = ['*']
instead of 
ALLOWED_HOSTS = [ ]

or you get the following error:
w[Wed Oct 18 20:05:00.978014 2017] [wsgi:error] [pid 12:tid 140541544236800] Invalid HTTP_HOST header: '10.0.0.1:8000'. You may need to add u'10.0.0.1' to ALLOWED_HOSTS.

Or at least add documentation to tell the user to change this according to:
https://docs.djangoproject.com/en/1.11/ref/settings/#allowed-hosts

### PR DESCRIPTION
**Pull request by kkuehl**

### CODE REVIEW

**Comment by jschwartzentruber** [comment]
These are both configuration specific, so I'd rather update `README.md` than make the defaults very permissive. We could also put comments in `settings.py` with your suggestions.

I didn't know about `DATA_UPLOAD_MAX_MEMORY_SIZE` and I'm surprised we haven't hit that yet since the upgrade. `ALLOWED_HOSTS` at least has a very clear error message if misconfigured.

**Comment by kkuehl** [comment]
Reverted settings.py changes and added section to README.md

File: README.md [hunk 1]
```diff
@@ -104,6 +104,20 @@ This .htpasswd file can be stored anywhere on your hard drive.
 Your Apache AuthUserFile line should be updated to reflect your path.
 See examples/apache2/default.vhost for an example
 
+### Important changes in settings.py
+It is important that you edit FuzzManager/server/settings.py and adjust the following variables according to your needs.
+
+ALLOWED_HOSTS = []
```
Comment (jschwartzentruber) [inline README.md]: This should be indented 4 spaces to be formatted properly.

File: README.md [hunk 2]
```diff
@@ -104,6 +104,20 @@ This .htpasswd file can be stored anywhere on your hard drive.
...
+
+See [ALLOWED_HOSTS documentation](https://docs.djangoproject.com/en/1.11/ref/settings/#allowed-hosts)
+
+
+You may also want to increase the maximum size in bytes allowed in a request body. The default of 2.5MB may not be enough
+in some cases by adding the following variable.
+
+DATA_UPLOAD_MAX_MEMORY_SIZE = <YOUR VALUE HERE>
```
Comment (jschwartzentruber) [inline README.md]: Ditto here. `<YOUR VALUE HERE>` doesn't actually show in the markdown preview.

**Comment by kkuehl** [comment]
Fixed.

**Pull request merged**